next/jest.config.js

```diff
@@ -12,6 +12,10 @@ const customJestConfig = {
   setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
   // if using TypeScript with a baseUrl set to the root directory then you need the below for alias' to work
   moduleDirectories: ['node_modules', '<rootDir>/'],
+  moduleNameMapper: {
```

Comment:
this deserves an explaining comment above and perhpas a mention in the docs as well